cpp/include/cuspatial/shapefile_reader.hpp

```diff
@@ -45,6 +46,7 @@ namespace cuspatial {
  **/
 std::vector<std::unique_ptr<cudf::column>> read_polygon_shapefile(
   std::string const& filename,
+  const bool reversed                 = false,
```

Comment:
I'm not sure if we can specify `CLOCKWISE` or `COUNTER_CLOCKWISE` here because we don't necessarily know the order of the polygons that are about to be read. That's why I just went with reversed.

Reply:
It's a parameter. The caller tells you the order. Once on inner rings you are just alternating.

Reply:
Er, right. :D Your suggested improvements make me feel a lot better about my contribution, rather than using a global.

Reply:
I like the refactor according to your suggestion, the code feels cleaner. I still worry though because we don't explicitly know the winding of the rings as-input: if a shapefile happens to have CLOCKWISE wound rings and we specify CLOCKWISE as the winding, they will be the opposite. I think I will put in the docs that it is assumed that all input shapefiles will natively have COUNTER_CLOCKWISE ordering.